nmap-service-probes

```diff
@@ -15473,3 +15473,22 @@ Probe TCP VersionRequest q|VERSION|
 ports 3310
 rarity 8
 match clam m|^ClamAV ([\w.]+)/(\w+)/(.+)$| p/ClamAV/ v/$1 ($2)/ i/AV definitions updated on:$3/
+
```

Comment:
I don't see why not! The system didn't experience any problems. I'll be honest, since I'm still a little inexperienced submitting new services probes for these mainframe items I wasn't aware you could do that (update a previous probe to match a different service). And I'm not entirely sure how to go about doing that so I'll wait to see how you change it or if you can advise and I can make the change and re-commit.